docs/DeveloperGuide.md

```diff
@@ -288,109 +286,155 @@ Priorities:
+| `* * *`  | Wedding organizer          | add and categorize vendors                         | keep track of service providers for each wedding                              |
+| `* * *`  | Wedding organizer          | view stakeholders related to each specific wedding | easily check their schedules                                                  |
+| `* * *`  | Wedding organizer          | update client preferences easily                   | accommodate last-minute changes and keep everything up to date                |
+| `* * *`  | Wedding organizer          | delete profiles and roles                          | remove outdated or incorrect information                                      |
+| `* * *`  | Wedding organizer          | quickly search for specific contacts or vendors    | access critical information without delays                                    |
+| `* * *`  | Wedding organizer          | filter contacts based on roles                     | retrieve contacts of stakeholders involved in a wedding or of a specific role |
+| `* *`    | Wedding organizer          | archive completed weddings                         | focus on current and upcoming events without clutter                          |
+| `* *`    | Wedding organizer          | track expenses for each wedding                    | manage the overall wedding budget efficiently                                 |
+| `* *`    | Wedding organizer          | set reminders for important tasks or deadlines     | ensure critical milestones are not missed                                     |
+| `* *`    | Wedding organizer          | track RSVPs from clients                           | keep an accurate guest count for each wedding                                 |
+| `* *`    | Wedding organizer          | generate reports on completed tasks                | review progress and share updates with clients                                |
+| `* *`    | Wedding organizer          | assign tasks to team members                       | delegate responsibilities and track progress efficiently                      |
+| `*`      | Wedding organizer          | upload important documents (e.g., contracts)       | access them quickly during planning                                           |
+| `*`      | Wedding organizer          | send automated reminders to vendors and clients    | ensure they stay informed of upcoming deadlines                               |
+| `*`      | Wedding organizer          | set up recurring tasks for common preparations     | avoid manually creating the same tasks for each event                         |
 
 ### Use Cases
 
 (For all use cases below, the **System** is *Bridal Boss* and the **Actor** is the *Wedding Organizer*.)
 
-#### **Use Case: Add a New Vendor to a Wedding**
+#### **Use Case: Add a Contact**
```

Comment:
should we refer them to the user guide for the command?

Reply:
howw hahah

Reply:
> howw hahah, bc in tht case everyth can be referred to the UG too

can link it at the top saying refer to all commands in the user guide?

Reply:
okies done!